src/pharmpy/modeling/odes.py

```diff
@@ -206,6 +206,84 @@ def set_first_order_elimination(model: Model):
     return model
 
 
+def add_bioavailability_statement(model: Model, add_parameter: bool = True):
+    """Add bioavailability statement for the dose compartment(s) of the model.
+    Can be added as a new parameter or simply as a number.
+
+    Parameters
+    ----------
+    model : Model
+        Pharmpy model
+    add_parameter : bool
+        Add new parameter representing bioavailability or not
+
+    Return
+    ------
+    Model
+        Pharmpy model object
+
+    Examples
+    --------
+    >>> from pharmpy.modeling import *
+    >>> model = load_example_model("pheno")
+    >>> model = add_bioavailability_statement(model)
+
+    See also
+    --------
+    remove_bioavailability
+
+    """
+    odes = model.statements.ode_system
+    if odes is None:
+        raise ValueError(f'Model {model.name} has no ODE system')
+
+    dose_comp = odes.dosing_compartment[0]
+    bio = dose_comp.bioavailability
+
+    if isinstance(bio, sympy.Number):
+        # Bio not defined
+        if add_parameter:
+            model, bio_symb = _add_parameter(model, 'BIO', init=float(bio))
+            ass = Assignment(sympy.Symbol('F'), bio_symb)
```

Comment:
If you use `Assignment.create()` it will automatically make it into a `sympy.Symbol` (same for numbers), see: https://github.com/pharmpy/pharmpy/blob/f357a01c2242be765dc93ddd7b94358d153f88fd/src/pharmpy/model/statements.py#L76-L89

Reply:
It now uses Assignment.create() instead.